content/admin/installation/installing-github-enterprise-server-on-google-cloud-platform.md

```diff
@@ -27,7 +27,7 @@ Before launching {% data variables.product.product_location_enterprise %} on Goo
 {% data variables.product.prodname_ghe_server %} is supported on the following Google Compute Engine (GCE) machine types. For more information, see [the Google Cloud Platform machine types article](https://cloud.google.com/compute/docs/machine-types).
 {% if currentVersion != "free-pro-team@latest" %}
 | High-memory   |
-  ------------- |
+| ------------- |
```

Comment:
This is the only "functional" change, as it broke on both the English and translated documents